REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
The requirements of ZMQ 0.3.3 and 0.3.4 are the same, so any time both are possible, Pkg will use the larger of them.